.emacs.d/lisp/code/languages/csharp.el

```diff
@@ -0,0 +1,91 @@
+;;  -*- lexical-binding: t; -*-
+
+;;; The following keybindings are available when editing C# files using omnisharp:
+;; C-RET: Do a refactoring at point. Prompt for which refactoring in the minibuffer.
+;; M-RET: Same as C-RET
+;; C-c o r: Intelligently rename the symbol at point, throughout project
+;; M-.: Jump to definition of symbol at point
+;; C-.: Find usages of symbol at point
+;; C->: Find implementations of symbol at point
+;; C-M-\\: Reformat the region
+
+
+(defun kotct/omnisharp-install-server ()
+  "Install omnisharp-roslyn, or, if using Arch Linux, make sure it is installed
+and then install the correct wrapper."
+  (interactive)
+  (let ((os (shell-command-to-string "lsb_release -ds")))
```

Comment:
What happens if this `let` fails? E.g. `lsb_release` doesn't exist.

Reply:
You will get an error message from the shell. Regardless, `os` won't be equal to the string indicating Arch.

Reply:
Okay. Will this break, (i.e. throw an error and bring up an error buffer) or just complain into `*Messages*`?  (I'm not familiar enough with this to be able to know off the top of my head.)

Reply:
Nothing will happen. It will just return a different string. Just do `M-: (shell-command-to-string "foo")` in emacs right now and you'll see what it does.

Reply:
Gotcha.  Thanks for the clarification.

(Throwback to how I have a command called `foo` lmao)

Reply:
lol, RIP my demo